### DISCUSSION TITLE
Embeddable player is missing

### DISCUSSION THREAD

**Discussion by ncfernan** [AzuraCast/AzuraCast#5964] [Q&A]
After 7 months and 541 changes behind, I started a new VM with the latest stable release of Azuracast. When the page I use to show the history opened, the embedded player disappeared and an error 500 is shown in its place.

My page is simple, and the code I use to show the player is quite straightforward:

```
  | <div id="div_player" class="div_player_class">
  | <iframe id="player" class="iframe_player_class" src="https://stream.ncfradio.com/public/mix/embed" frameborder="0" allowtransparency="false">
  | </iframe>
  | </div>
```

![image](https://user-images.githubusercontent.com/10008844/209567505-58c53054-043e-496b-99dd-5376e109bd5a.png)

My question is: where did the good and old embeddable player go?

**Comment by BusterNeece** [comment]
@ncfernan It's still there...we didn't remove it. It just doesn't think your station has public pages enabled or some other setting isn't right (i.e. the station shortname changed, which would change that URL.)

**Reply by ncfernan** [reply]
@BusterNeece you're right (as always). The public pages was disabled. Now it shows nicely. Thank you
